### PR TITLE
docs(app): Improve load model docs page

### PR DESCRIPTION
**Pull request by JaimeTorrealba**

### CODE REVIEW

File: docs/examples/load-models.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ For this guide we are going to focus on loading gLTF (GL Transmission Format) mo
 
 There are several ways to load models on TresJS:
 
+::: warning
+Please note that the examples above we use top level await, make sure you wrap it with a [Suspense](https://vuejs.org/guide/built-ins/suspense.html#suspense) component. See Suspense for more information. .
+:::
+
```
Comment (alvarosabu) [inline docs/examples/load-models.md:16]: 🙏🏻 @JaimeTorrealba

**Review by alvarosabu — CHANGES_REQUESTED** [review]
Screenshot 2024-01-12 at 16 51 14

Now that the suspense is gone, the highlighted lines are incorrect in most of the doc, could you fix them please?

File: docs/examples/load-models.md (hunk 1 above)
Comment (JaimeTorrealba) [reply docs/examples/load-models.md:16]: Done

**Review by alvarosabu — APPROVED** [review]

**Comment by JaimeTorrealba** [comment]
@alvarosabu this is also ready, please let me know

**Pull request merged**